### PR TITLE
ProfitBricks API v4 support & bug fixing

### PR DESCRIPTION
**Pull request by nurfet-becirevic**
* Support for ProfitBricks API v4:
  * Add `profitbricks_image_alias` flag & support for image aliases
  * Add new location, `us/ewr`
* Fix issue with extra `/` in the API url
* Fix volume availability zone issue

### CODE REVIEW

**Comment by nurfet-becirevic** [comment]
I signed it.

File: CHANGES.next.md [hunk 1]
```diff
@@ -9,7 +9,11 @@
 - Windows benchmarks can now be run from linux controllers
 
 ###Enhancements:
+- Support for ProfitBricks API v4:
+  - Add `profitbricks_image_alias` flag and support for image aliases
+  - Add new location, `us/ewr`
 
 ###Bug fixes and maintenance updates:
-- Moved GPU-related specs from GceVmSpec to BaseVmSpec
```
Comment (s-deitz) [inline CHANGES.next.md]: Keep this note.

File: perfkitbenchmarker/providers/profitbricks/__init__.py [hunk 2]
```diff
@@ -13,4 +13,4 @@
 # limitations under the License.
 """ProfitBricks cloud provider implementation."""
 
-PROFITBRICKS_API = 'https://api.profitbricks.com/cloudapi/v3/'
+PROFITBRICKS_API = 'https://api.profitbricks.com/cloudapi/v4'
```
Comment (s-deitz) [inline perfkitbenchmarker/providers/profitbricks/__init__.py]: I'm unable to run profitbricks at this time. Which benchmarks have you tested this change on?

Comment (nurfet-becirevic) [reply perfkitbenchmarker/providers/profitbricks/__init__.py]: I tested iperf and ping. The example form README should work.
```
./pkb.py --cloud=ProfitBricks --machine_type=Small --benchmarks=iperf
```

File: CHANGES.next.md (hunk 1 above)
Comment (nurfet-becirevic) [reply CHANGES.next.md]: The note is back.

**Pull request merged**